### PR TITLE
Docs improvement

### LINKED ISSUE #28: Extend the readme
Hey @GrahamCampbell I actually am not 100% sure what this does, and how it does what it does. 😄 Could you maybe add a section to explain what it does and also another one with actual usage examples?

From the tests I gathered that it is input sanitisation, but to what extend, and what is actually possible with it? Some more real code examples would be great, thanks.

### PR DESCRIPTION
**Pull request by lukasoppermann**
I think this could make it easier to understand. I would also add an example to clarify what the `xss` boolean does, as well as showing the default values from the config.

If you are fine with this I can create an additional PR with those changes.

I think that users who are not comfortable with tests, etc. will not look into those, and you do not want to digg through tests just to figure out what a package does.

Worst scenario, people just don't use the package, which would be sad, as it is a really good one.

This would fix #28

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,9 @@
 Laravel Binput
 ==============
 
-Laravel Binput was created by, and is maintained by [Graham Campbell](https://github.com/GrahamCampbell), and is an input protector for [Laravel 5](http://laravel.com). It utilises my [Laravel Security](https://github.com/GrahamCampbell/Laravel-Security) package. Feel free to check out the [change log](CHANGELOG.md), [releases](https://github.com/GrahamCampbell/Laravel-Binput/releases), [license](LICENSE), and [contribution guidelines](CONTRIBUTING.md).
+Laravel Binput was created by, and is maintained by [Graham Campbell](https://github.com/GrahamCampbell), and is an input protector for [Laravel 5](http://laravel.com) that strips unwanted elements, e.g. `<script>` tags from any input you receive. It utilises my [Laravel Security](https://github.com/GrahamCampbell/Laravel-Security) package.
```
Comment (GrahamCampbell) [inline README.md]: Hmm, this description isn't quite true.

Comment (lukasoppermann) [inline README.md]: I might have misunderstood something. 😉 Please let me know what I should change.

Comment (GrahamCampbell) [inline README.md]: It prevents them doing damage rather than removing all traces of them.

**Comment by lukasoppermann** [comment]
Will change it once I am back at my notebook.

**Comment by lukasoppermann** [comment]
Hey @GrahamCampbell, I updated the text, do you think this now reflects the reality better?

**Comment by lukasoppermann** [comment]
Hey @GrahamCampbell what do you think, can this be merged, or would you need something change first?

**Comment by GrahamCampbell** [comment]
Thanks. I'll tweak it myself now.

**Pull request merged**